src/pages/dashboard/components/header/assets/logo.svg

```diff
@@ -1,5 +1,5 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="628" height="348" viewBox="0 0 628 348">
-  <g fill="#FFF" fill-rule="evenodd">
+  <g fill="#FFF" fillRule="evenodd">
```

Comment:
I'm not sure this is necessary on a raw SVG?

Also, we should probably get rid of this and use the newer CC Logo component.

Reply:
React was throwing errors because of this 🤷

On the other hand is this the oficial logo? I think we were going to have both, light and dark @trent-boyd